classes/player/player.gd

```diff
@@ -1387,7 +1390,7 @@ func off_ground():
 	cancel_ground = true
 
 
-func recieve_health(amount):
+func receive_health(amount):
```

Comment:
Please try to limit small changes like this to their own PR. This almost broke a different script, since it left a function call hanging without its respective definition.

It's not a huge issue, just a word of advice. ^^;;